wpk/run.sh

```diff
@@ -2,13 +2,14 @@
 
 DIRECTORY="wazuh"
 REPOSITORY="https://github.com/wazuh/wazuh.git"
-BRANCH=$1
-JOBS=$2
-OUT_NAME=$3
-CHECKSUM=$4
-PKG_NAME=$5
+BRANCH=${1}
+JOBS=${2}
+OUT_NAME=${3}
+CHECKSUM=${4}
+DEFAULT_INSTALLATION_PATH=${5}
```

Comment:
Same here, change it to `INSTALLATION_PATH`.

Reply:
Fixed https://github.com/wazuh/wazuh-packages/pull/408/commits/266a8eff05ec92416c5eee4eb6f8adbfbf3a07ea